### PR TITLE
Throw error when tsconfig.json has excludes without exclude property

### LINKED ISSUE #4957: Give a helpful error when tsconfig.json has "excludes" without "exclude" property
There's definitely always going to be people who type in 'excludes' instead of 'exclude' (https://github.com/Microsoft/TypeScript/issues/4939#issuecomment-1430313820), so let's actually be helpful in that case.

In addition, in a language service context, maybe we should still figure out what the user means and just pretend they wrote 'exclude' in addition to showing the error.

### PR DESCRIPTION
**Pull request by YuichiNukiyama**
Fixes #4957

### CODE REVIEW

File: src/compiler/diagnosticMessages.json [hunk 1]
```diff
@@ -2243,6 +2243,10 @@
         "category": "Error",
         "code": 5062
     },
+    "Option 'excludes' is invalid. You should use 'exclude'": {
```
Comment (mhegazy) [inline src/compiler/diagnosticMessages.json]: `Unknown option 'excludes'. Did you mean 'exclude'`

File: src/compiler/diagnosticMessages.json [hunk 2]
```diff
@@ -2243,6 +2243,10 @@
         "category": "Error",
         "code": 5062
     },
+    "Option 'excludes' is invalid. You should use 'exclude'": {
+        "category": "Error",
+        "code": 5063
```
Comment (mhegazy) [inline src/compiler/diagnosticMessages.json]: i would move this to the 6000 range with the other options messages

**Comment by mhegazy** [comment]
thanks @YuichiNukiyama, 2 small comments, but other than that we are good to go.

**Comment by YuichiNukiyama** [comment]
@mhegazy Thank you for your review. I did it.

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**